src/electionguard/election.py

```diff
@@ -27,6 +28,18 @@ class Configuration:
     This can also be seen as the maximum ballots where a selection on a ballot can only have one vote.
     """
 
+    write_in_size: PaddedDataSize = field(default=PaddedDataSize.Bytes_64)
```

Comment:
These are names I came up with to describe these fields and what they do that I thought would be useful.

It could be strongly argued for these fields to be called.

`selection_extended_data_size`
`contest_extended_data_size`

Reply:
The name you have seems solid, I consider it a good sign when names match descriptions, which this does